pkgs/native_assets_cli/example/build/linked_library/lib/math.dart

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+// AUTO GENERATED FILE, DO NOT EDIT.
+//
+// Generated by `package:ffigen`.
+// ignore_for_file: type=lint
+import 'dart:ffi' as ffi;
+
+@ffi.Native<ffi.Int32 Function(ffi.Int32, ffi.Int32)>(symbol: 'math_add')
```

Comment:
We should probably not generate this, because we have `add.c` invoking `math.c`. It's a bit weird to invoke `math.c` then also from Dart. (We risk accidentally first invoking `math_add` from Dart and loading the dylib that way instead of having the C dynamic linker load the dylib.)